grr/server/grr_response_server/gui/api_plugins/timeline.py

```diff
@@ -112,17 +112,22 @@ def Handle(
       raise ValueError(message)
 
     filename = f"timelines_{hunt_id}.zip"
-    content = self._Generate(hunt_id)
+    content = self._Generate(hunt_id, args.format)
     return api_call_handler_base.ApiBinaryStream(filename, content)
 
-  def _Generate(self, hunt_id: Text) -> Iterator[bytes]:
+  def _Generate(
+      self,
+      hunt_id: Text,
+      format: ApiGetCollectedTimelineArgs.Format
```

Comment:
Try to use trailing commas in such cases—I am pretty sure that our internall formatter would move the paren to this line without it. Also, it makes the diffs nicer if you add more arguments.

Reply:
Makes sense. Thank you!